proxy/src/schema/mod.rs

```diff
@@ -73,6 +82,45 @@ impl Mutation {
             metadata: meta.into(),
         })
     }
+
+    fn register_project(
+        ctx: &Context,
+        domain: String,
```

Comment:
Considering https://github.com/radicle-dev/radicle-registry/issues/152, shouldn't we here also validate the `domain` to be `.rad`. Or better, to not receive `domain` since only `.rad` will be supported?

Reply:
Not yet certain where the best way is to make the cut off, we want to support multiple domains down the road, maybe even sooner with the advent of orgs.